modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -0,0 +1,120 @@
+|xref:anypoint-cli::secrets-manager.adoc#secret-keystore-list[secrets-mgr:keystore:list]| Lists all keystore secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-keystore-modify[secrets-mgr:keystore:modify]| Modifies a keystore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-keystore-replace[secrets-mgr:keystore:replace]| Replaces an existing keystore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-create[secrets-mgr:truststore:create]| Creates a new truststore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-describe[secrets-mgr:truststore:describe]| Shows details of a truststore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-list[secrets-mgr:truststore:list]| Lists all truststore secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-modify[secrets-mgr:truststore:modify]| Modifies a truststore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-truststore-replace[secrets-mgr:truststore:replace]| Replaces an existing truststore secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-create[secrets-mgr:tls-context:mule:create]| Creates a new Mule TLS Context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-describe[secrets-mgr:tls-context:mule:describe]| Shows details of a Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-list[secrets-mgr:tls-context:mule:list]| Lists all Mule TLS context secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-modify[secrets-mgr:tls-context:mule:modify]| Modifies a Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-context-replace[secrets-mgr:tls-context:mule:replace]| Replaces an existing Mule TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-create[secrets-mgr:tls-context:flex-gateway:create]| Creates a new Flex Gateway TLS Context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-describe[secrets-mgr:tls-context:flex-gateway:describe]| Shows details of a Flex Gateway TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-list[secrets-mgr:tls-context:flex-gateway:list]| Lists all Flex Gateway TLS context secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-modify[secrets-mgr:tls-context:flex-gateway:modify]| Modifies a Flex Gateway TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-replace[secrets-mgr:tls-context:flex-gatway:replace]| Replaces an existing Flex Gateway TLS context secret
+
+
+|===
+
+// end::summary[]
+
+// tag::commands[]
+
+[[secret-group-create]]
+== secrets-mgr:secret-group:create
+
+----
+> secrets-mgr:secret-group:create [flags] 
+----
+
+This Command creates a new secret group with the name specified in `--name`.
+
+Prompt the `--downloadable` flag if you want this group to be downloadable by members of your organization.
```

Comment:
'--downloadable' flag is not to enable downloading of secrets for org members.  

When creating API Manager Proxy (Mule Gateway), API Manager will list only the secret groups that are marked as "downloadable" from which TLS context can be selected.  Since Proxy jar artifacts that embeds the TLS context secrets can be downloaded by customers , this "downloadable" opt-in flag is to ensure that the customer is aware that the secrets in this secret group can be included in the downloaded artifact.